src/odemis/gui/win/acquisition.py

```diff
@@ -643,7 +643,7 @@ def __init__(self, parent, orig_tab_data):
 
         # The pattern to use for storing each tile file individually
         # None disables storing them
-        self.filename_tiles = create_filename(self.conf.last_path, "{datelng}-{timelng}-overview",
+        self.filename_tiles = create_filename(self.conf.pj_last_path, "{datelng}-{timelng}-overview",
```

Comment:
It's not that simple because this window can be called from the secom or delphi too, which don't have a `pj_last_path`. In such case, `last_path` is good. That's why OOP should help with, but I'm not sure how to handle it here nicely. An alternative would be to pass it as an argument to the OverviewAcquisitionDialog. Another way would be to check if it's enzel/meteor or other (but it sounds pretty easy to break).